ptf

```diff
@@ -231,6 +235,9 @@ be subtracted from the result by prefixing them with the '^' character.  """
 
     parser.add_argument("--pypath", dest="pypath", action="append")
 
+    parser.add_argument("--packet-manipulation-module", type=str,
```

Comment:
Can you add some short form? Like `-pmm`?

Reply:
Sure, it can be added.